luxon/src/impl/conversions.js

```diff
@@ -0,0 +1,161 @@
+import {
+  integerBetween,
+  isLeapYear,
+  timeObject,
+  daysInYear,
+  daysInMonth,
+  weeksInWeekYear,
+  isInteger,
+} from './util.js';
+import Invalid from './invalid.js';
+
+const nonLeapLadder = [0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334];
+const leapLadder = [0, 31, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335];
+
+function unitOutOfRange(unit, value) {
+  return new Invalid(
+    'unit out of range',
+    `you specified ${value} (of type ${typeof value}) as a ${unit}, which is invalid`,
+  );
+}
+
+function dayOfWeek(year, month, day) {
+  const js = new Date(Date.UTC(year, month - 1, day)).getUTCDay();
+  return js === 0 ? 7 : js;
+}
+
+function computeOrdinal(year, month, day) {
+  return day + (isLeapYear(year) ? leapLadder : nonLeapLadder)[month - 1];
+}
+
+function uncomputeOrdinal(year, ordinal) {
+  const table = isLeapYear(year) ? leapLadder : nonLeapLadder;
+  const month0 = table.findIndex((i) => i < ordinal);
+  const day = ordinal - table[month0];
+  return { month: month0 + 1, day };
+}
```

Comment:
- [ ] Please as required by the previous review, you should use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
@moise10r Thank you for your comments. All the files inside the luxon folder belong to the luxon library, it is a dependency. I imported it via npm, complying with the requirement of the project. I moved it in the root directory in order to reference it and use it on ghPage ( Thanks to the sound advice of @TedLivist ). The comment made about the use of the arrow function is related to the navigation file - navigation.js- ( code that I created).  From a practical and technical point of view, this is not feasible. Please consider reviewing your recommendation for this part of the code.